src/Datum.ts

```diff
@@ -275,18 +280,81 @@ export function getOrd<A>(O: Ord<A>): Ord<Datum<A>> {
  * `Apply` semigroup
  *
  * @since 2.0.0
+ * 
+ * Note: Semantics changed in 4.0.0 to match new Apply instance
```

Comment:
This is may be the only breaking change so far. Not sure your thoughts on breaking changes, but you did suggested a major version bump so I figured it was worth discussing.

Given the datum mega-instance is deprecated in the PR and the standalone Apply instance has different behavior, it seemed warranted to "fix" this very clearly "Apply"-based instance factory.

Reply:
For now let's follow the fp-ts path for deprecating changes. Add the @deprecated doc tag to `getApplySemigroup` with a note about its incoming change. Then create a new `getApplySemigroup2` with the new behavior.